modules/new_user/new_user.py

```diff
@@ -24,24 +24,31 @@ async def add_role(ctx, machon, year):
 	print(f"Gave {new_role} to {member}")
     
 
-async def remove_unassigned(member):
+async def switch_assigned(member):
 	'''removes the unassigned role from member'''
-	role = get(member.guild.roles, id = int(get_id("UNASSIGNED_ROLE_ID")))
+	#remove unassigned role
+	role = get(member.guild.roles, id=get_id("UNASSIGNED_ROLE_ID"))
 	await member.remove_roles(role)
+
+	#add assigned role
+	role = get(member.guild.roles, id=get_id("ASSIGNED_ROLE_ID"))
+	await member.add_roles(role)
+
 	print(f"Removed Unassigned from {member}")
     
     
 async def change_nick(member, first_name, last_name):
 		'''changes the nick name of the member to their full name'''
-		name = first_name.capitalize() + ' ' + last_name.capitalize()
+		name = first_name.strip(',').capitalize() + ' ' + last_name.strip(',').capitalize()
 		await member.edit(nick = name)
 		print(f"Set {member} nick to {name}")
 
+		
 def get_id(name):
```

Comment:
The CSV library should provide a way to read the CSV without you having to parse it. Try to use that